src/frontend/SentryHandler/index.tsx

```diff
@@ -19,6 +19,7 @@ const SentryHandler = () => {
             ? devSentryDsn
             : prodSentryDsn
       },
+      // @ts-expect-error - not sure why is this failling
       reactInit
```

Comment:
might be a mismatch between the `@sentry/electron` and `@sentry/react` package versions?

Reply:
I will check

Reply:
I think maybe we should update Sentry to latest packages and do the SDK migration on another PR.
I was looking into the migration guide and the new docs and they diverge in some points.
The electron one says its possible to use the Sentry.init to start a second SDK, but on the latest version its not possible anymore because it just accepts the options so on the frontend we would use only the React SDK instead of also importing from sentry/electron/renderer.
Since I don't really know how to test this lets keep it like this on this PR.

But that complain that I added the expect error its because it says that the type of ReactInit mismatches what Sentry.init is expecting.